<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgConvertColorSpace</class>
 <widget class="QWidget" name="WdgConvertColorSpace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Colorspace Conversion</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="KisColorSpaceSelector" name="colorSpaceSelector"/>
   </item>
   <item>
    <widget class="QGroupBox" name="grpIntent">
     <property name="toolTip">
      <string/>
     </property>
     <property name="title">
      <string>&amp;Rendering Intent</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioPerceptual">
        <property name="toolTip">
         <string>For images</string>
        </property>
        <property name="whatsThis">
         <string>Hue hopefully maintained (but not required),
lightness and saturation sacrificed to maintain
the perceived color. White point changed to
result in neutral grays. Intended for images.</string>
        </property>
        <property name="text">
         <string>Perceptual</string>
        </property>
        <property name="shortcut">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioRelativeColorimetric">
        <property name="whatsThis">
         <string>Within and outside gamut; same as Absolute
Colorimetric. White point changed to result in
neutral grays.

If adequate table is present in profile,
then, it is used. Else reverts to perceptual
intent.</string>
        </property>
        <property name="text">
         <string>Relative colorimetric</string>
        </property>
        <property name="shortcut">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="radioSaturation">
        <property name="toolTip">
         <string>Best for graphs and charts</string>
        </property>
        <property name="whatsThis">
         <string>Hue and saturation maintained with lightness
sacrificed to maintain saturation. White point
changed to result in neutral grays. Intended for
business graphics (make it colorful charts,
graphs, overheads, ...)

If adequate table is present in profile,
then, it is used. Else reverts to perceptual
intent.</string>
        </property>
        <property name="text">
         <string>Saturation</string>
        </property>
        <property name="shortcut">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="radioAbsoluteColorimetric">
        <property name="toolTip">
         <string>Best for spot colors</string>
        </property>
        <property name="whatsThis">
         <string>Within the destination device gamut; hue,
lightness and saturation are maintained. Outside
the gamut; hue and lightness are maintained,
saturation is sacrificed. White point for source
and destination; unchanged. Intended for spot
colors (Pantone, TruMatch, logo colors, ...)</string>
        </property>
        <property name="text">
         <string>Absolute colorimetric</string>
        </property>
        <property name="shortcut">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkBlackpointCompensation">
     <property name="text">
      <string>Use Blackpoint &amp;Compensation</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkAllowLCMSOptimization">
     <property name="toolTip">
      <string>Krita tries to automatically check or uncheck this based on the information it can get from the profile you are converting from and the profile you are converting to.</string>
     </property>
     <property name="text">
      <string>Allow Little CMS optimizations (uncheck when using linear light RGB or XYZ)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisColorSpaceSelector</class>
   <extends>QWidget</extends>
   <header>widgets/kis_color_space_selector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
